src/protocols/filehandler.cpp

```diff
@@ -45,7 +45,7 @@ bool FileHandler::startRequest(const QUrl &url, RequestOptions options)
         for (unsigned i = 0; i < dir.count(); ++i)
         {
             // Add link to page.
-            page += QString("=> file://%1 %2\n")
+            page += QString(tr("=> file://%1 %2\n"))
```

Comment:
i don't think *any* translation should happen here, this displays either a URI (*must be* passthrough and is is dynamic) or a directory name (which also should be pass through)

Everything else here is defined and the string `"=> file://%1 %2"` must not change between translations

Reply:
When two text elements follow each other, localizers should be able to add LTR/RTL control characters between them to prevent unintended text rearrangement in bidirectional locales.

Reply:
Huh, i see. Can you make an example how that would look like?

Reply:
Something like "=> %1 ↱%2", where the arrow is an LTR marker.

As for your other question, there is no single way. Usually, the app creates a `QTranslator` for the locale file the user chose in the settings, and installs it dynamically.

Reply:
Ah, okay. Thanks for clarification! I'll try to get dynamic translation switching then :)